.circleci/config.yml

```diff
@@ -1,13 +1,21 @@
 version: 2
 
 jobs:
-  build:
+  build_libgo_cosmwasm:
     docker:
       - image: circleci/rust:1.41-stretch
     steps:
       - run: rustup toolchain install nightly
+      - run:
+          name: Show version information
+          command: rustc --version; cargo --version; rustup --version
       - checkout
-      - run: make build-rust
+      - run:
```

Comment:
Nice one. Forgot there were any rust tests...

Reply:
There were none. I wrote the first one today